roles/consul/defaults/main.yml

```diff
@@ -61,30 +61,31 @@ consul_systemd_limit_nofile: 65536
 consul_systemd_unit_path: "/lib/systemd/system"
 
 ### Log user, group, facility
-syslog_user: "{{ lookup('env','SYSLOG_USER') | default('root', true) }}"
-syslog_group: "{{ lookup('env','SYSLOG_GROUP') | default('adm', true) }}"
-consul_log_level: "{{ lookup('env','CONSUL_LOG_LEVEL') | default('INFO', true) }}"
-consul_log_rotate_bytes: "{{ lookup('env','CONSUL_LOG_ROTATE_BYTES') | default(0, true) }}"
-consul_log_rotate_duration: "{{ lookup('env','CONSUL_LOG_ROTATE_DURATION') | default('24h', true) }}"
-consul_log_rotate_max_files: "{{ lookup('env','CONSUL_LOG_ROTATE_MAX_FILES') | default(0, true) }}"
-consul_syslog_enable: "{{ lookup('env','CONSUL_SYSLOG_ENABLE') | default(false, true) }}"
-consul_syslog_facility: "{{ lookup('env','CONSUL_SYSLOG_FACILITY') | default('local0', true) }}"
-consul_configure_syslogd: "{{ lookup('env','CONSUL_CONFIGURE_SYSLOGD') | default(false, true) }}"
+syslog_user: "{{ lookup('env', 'SYSLOG_USER') | default('root', true) }}"
+syslog_group: "{{ lookup('env', 'SYSLOG_GROUP') | default('adm', true) }}"
+consul_log_level: "{{ lookup('env', 'CONSUL_LOG_LEVEL') | default('INFO', true) }}"
+consul_log_rotate_bytes: "{{ lookup('env', 'CONSUL_LOG_ROTATE_BYTES') | default(0, true) }}"
+consul_log_rotate_duration: "{{ lookup('env', 'CONSUL_LOG_ROTATE_DURATION') | default('24h', true) }}"
+consul_log_rotate_max_files: "{{ lookup('env', 'CONSUL_LOG_ROTATE_MAX_FILES') | default(0, true) }}"
+consul_syslog_enable: "{{ lookup('env', 'CONSUL_SYSLOG_ENABLE') | default(false, true) }}"
+consul_syslog_facility: "{{ lookup('env', 'CONSUL_SYSLOG_FACILITY') | default('local0', true) }}"
+consul_configure_syslogd: "{{ lookup('env', 'CONSUL_CONFIGURE_SYSLOGD') | default(false, true) }}"
 
 ### Consul settings
-consul_datacenter: "{{ lookup('env','CONSUL_DATACENTER') | default('dc1', true) }}"
-consul_domain: "{{ lookup('env','CONSUL_DOMAIN') | default('consul', true) }}"
-consul_alt_domain: "{{ lookup('env','CONSUL_ALT_DOMAIN') | default('', true) }}"
+consul_datacenter: "{{ lookup('env', 'CONSUL_DATACENTER') | default('dc1', true) }}"
+consul_domain: "{{ lookup('env', 'CONSUL_DOMAIN') | default('consul', true) }}"
+consul_alt_domain: "{{ lookup('env', 'CONSUL_ALT_DOMAIN') | default('', true) }}"
 consul_node_meta: {}
-consul_iface: "\
-  {% if ansible_os_family == 'Windows' %}\
-    {{ lookup('env','CONSUL_IFACE') | default(ansible_interfaces[0].interface_name, true) }}\
-  {% else %}\
-    {{ lookup('env','CONSUL_IFACE') | default(ansible_default_ipv4.interface, true) }}\
-  {% endif %}"
-consul_node_role: "{{ lookup('env','CONSUL_NODE_ROLE') | default('client', true) }}"
-consul_recursors: "{{ lookup('env','CONSUL_RECURSORS') | default('[]', true) }}"
-consul_bootstrap_expect: "{{ lookup('env','CONSUL_BOOTSTRAP_EXPECT') | default(false, true) }}"
+consul_iface: >
```

Comment:
it looks like there was a small refactoring of the code here, although this is not in the description of the PR

testing of the 'dcs_type: consul' schema has not been automated yet, has there been an attempt to deploy a cluster of Type C schema manually?

Reply:
Yes, I had seen this optimization.

Based on my analysis, it appears to be correct and more readable.

However, it is indeed outside the scope of this PR. Could it be considered premature optimization?

In any case, it is true that we sorely lack tests for this particular part (Type C).

I will try to see if I can come up with an Ansible test code for this part (before and after the change) to verify the expected results.

Reply:
> However, it is indeed outside the scope of this PR. Could it be considered premature optimization ?

I am not against refactoring it is for the benefit of the project, but it is advisable to describe all the changes in the description of PR or create separate PR for different tasks, for example jinja spacing is one thing and refactoring is another PR.

> In any case, it is true that we sorely lack tests for this particular part (Type C).

I have already allocated several hours, but it has not worked out yet, there are difficulties in the fact that the dcs_type value cannot be redefined at the level of a separate molecule test since 'vars_files' is used for vars/main.yml

If you have any ideas, I will be grateful. But this is beyond the scope of the topic of this PR, you can discuss in the issue https://github.com/vitabaks/postgresql_cluster/issues/375